scripts/apptester/apps.yam

```diff
@@ -175,3 +175,18 @@ sparkdemo:
         posix:
           dependencies: [["common/build-scripts/build.sh", "."]]
           buildcmd: "bash -c \"./build.sh build\""
+
+profilingdemo:
+  name: "Profiling demo"
+  language:
+    c:
+      src: "profilingdemo/source/c"
+      platform:
+        posix:
+          buildcmd: "bash -c \"./build.sh build\""
```

Comment:
Threre is no dependencies for C Profiling Demo, it is completely contained in `profilingdemo/source/c` directory